### DISCUSSION TITLE
Charts not working when 'Time' metric applied

### DISCUSSION THREAD

**Discussion by agerrard92** [community/community#57605] [Projects and Issues]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues (The feature that allows you to track ideas, feedback, tasks, or bugs for work on GitHub. Not for any issue you are having).
- [X] Projects (For planning and tracking work on GitHub - not any open source or other type of project).

### Body

When trying to conduct analysis of our working patterns, we use charts to help see whether changes to our processes have made delivery quicker. When we create charts, they work fine. Until they are configured to use 'Time' on the X axis. We receive an Error: loading data unable to render chart. Any help would be v much appreciated. 

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by mtcA6** [comment]
experiencing the same issue
